monkey/infection_monkey/island_api_client/http_requests_facade.py

```diff
@@ -0,0 +1,99 @@
+# Retries improve reliability and slightly mitigates performance issues
```

Comment:
Move this comment down where it makes more sense. Probably line 35 or in that area. Putting it at the top of the file is misleading. Retrying is not what everything in this files does.

Reply:
Yeah, not sure how it got here, IDE probably auto-generated imports in between those lines :D